### PR TITLE
Update release.yaml

### PR DESCRIPTION
**Pull request by petesfrench**
## Done

- Updates release.yaml based on this [discouse post](https://discourse.ubuntu.com/t/jammy-jellyfish-release-notes/24668) 

## QA

- Compare against the [discouse post](https://discourse.ubuntu.com/t/jammy-jellyfish-release-notes/24668)

### CODE REVIEW

File: releases.yaml [hunk 1]
```diff
@@ -9,44 +9,50 @@ latest:
   past_eol_date: false
   release_notes_url: "https://discourse.ubuntu.com/t/impish-indri-release-notes/21951"
 lts:
-  slug: FocalFossa
+  slug: JammyJellyfish
+  name: "Jammy Jellyfish"
+  short_version: "22.04"
+  full_version: "20.04"
```
Comment (sowasred2012) [inline releases.yaml:15]:
```suggestion
  full_version: "22.04"
```

File: releases.yaml [hunk 2]
```diff
@@ -9,44 +9,50 @@ latest:
...
-previous_previous_lts:
-  name: "Xenial Xerus"
-  short_version: "16.04"
-  full_version: "16.04.7"
+openstack_lts:
+  slug: Yoga
 
 checksums:
   desktop:
+    "22.04": "update-me *ubuntu-22.04-desktop-amd64.iso"
```
Comment (sowasred2012) [inline releases.yaml:34]: Would it be better to mark this PR as blocked/don't merge until we can get the actual checksums for these?

**Review by sowasred2012 — CHANGES_REQUESTED** [review]
There's a typo in `full_version`:
Screenshot from 2022-04-05 14-16-21

File: releases.yaml (hunk 2 above)
Comment (petesfrench) [reply releases.yaml:34]: Can do, I just made building some of the pages a little clearer.

Comment (pmahnke) [reply releases.yaml:34]: if you don't have this merged, it is really hard to make sure all the logic works on the download pages.

**Review by sowasred2012 — APPROVED** [review]
LGTM - spoke to Peter and he explained why it's useful to have this with placeholders for the checksums, I hadn't realised.

**Pull request merged**